setup.py

```diff
@@ -4,7 +4,6 @@
 setuptools.setup(name='disbatch', version='2.0-RC3',
```

Comment:
We should bump the version here

Reply:
Sure. 2.1 (no -RCx)?

Reply:
RCx will confuse pip versioning probably. I tend to just set these to anticipated next version, even if there are commits after. 2.1 sounds fine. or 2.0.1, since no new real functionality has been added.